pkg/filepiece/fpiece.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package fpiece
+
+import (
+	"errors"
+	"io"
+	"os"
+)
+
+// Chunk - Section of data to be concurrently read
+type Chunk struct {
+	file       *os.File
+	start      int64
+	final      int64
+	currentPos int64
+}
+
+// seek enum
+const (
+	SeekStart   = iota // seek relative to the origin of the file
+	SeekCurrent = iota // seek relative to the current offset
+	SeekEnd     = iota // seek relative to the end
+)
```

Comment:
Just found these same constants defined in the `io` package: https://golang.org/pkg/io/#pkg-constants

So you can just use `io.SeekStart`, etc instead of defining your own constants.